### ISSUE TITLE
new selector for detector

### ISSUE THREAD

**Issue by carandraug** [MicronOxford/SpekCheck#65]
While writing the paper, there doesn't seem to be a good reason to not have detectors handled separately on the interface.  After speaking with Ian and Carina, I will be adding them.

**Comment by carandraug** [comment]
I have added support for detectors as a separate entity all over the place. I have named their spectrum property as 'qe' since that's what was used on their csv files (sensitivity was used for the eye).  Only missing now that the emission efficiency and transmission needs to check the detector.

**Comment by iandobbie** [comment]
Thanks David

The detectors were always a real afterthought. Adding them properly is a good idea.

Ian

Sent from my iPhone

On 12 Apr 2018, at 21:42, Carnë Draug <notifications@github.com<mailto:notifications@github.com>> wrote:


I have added support for detectors as a separate entity all over the place. I have named their spectrum property as 'qe' since that's what was used on their csv files (sensitivity was used for the eye). Only missing now that the emission efficiency and transmission needs to check the detector.
